### ISSUE TITLE
Duplicate Lotek-270 receiver

### ISSUE THREAD

**Issue by zcrysler** [jbrzusto/TO_DO#126]
James Wright has pointed out an overlapping receiver: Lotek-270 is in projects 32 and 64, is this one already on your list of duplicates?  If not can you mark one of them like you have the others?  They are both active at the same time and it seems the detection data has merged based on only 1 of the deployment files which he thinks is the wrong one.  If there's a problem with detections from these receivers, will data from them have to be re-run?

**Comment by jbrzusto** [comment]
So we need a way to automatically distinguish files coming from different Lotek receivers
that have the same serial number.

Perhaps the way is to have users set the "Site Code" field, as it is reported in the .DTA files and is presumably something users can set?  For two users with different Lotek receivers having the
same serial number, we'd give them each a distinct site code to set on their receiver, and then
process incoming files based on both serial number and site code.  Ideally, we'd ask them to put
a sticker on their receiver indicating that for use with motus, the site code **must be** set to **X**.
But I don't know enough about the Lotek receivers to know whether this is possible.

In this particular case, the files we have for Lotek-270 have these site codes:

Filename | Site Code
---|---
HOFT.DTA |  0001
hoftFINAL.DTA |  0001
MOU_03Dec16.DTA |  0270
MOU_14Mar2017.DTA |  0270
MOU_15May2017.DTA |  0270
MOU_16Apr2017.DTA |  0270
MOU_17Nov16.DTA |  0270
MOU_1Apr2017.DTA |  0270
MOU_21Jun2017.DTA |  0270
MOU_22Aug2017.DTA |  0270
MOU_22Mar2017.DTA |  0270
MOU_23Apr2017.DTA |  0270
MOU_24Jul2017.DTA |  0270
MOU_25Nov16.DTA |  0270
MOU_26Mar2017.DTA |  0270
MOU_27Mar2017.DTA |  0270
MOU_3May2017.DTA |  0270
MOU_4Mar17.DTA |  0270
MOU_8May2017.DTA |  0270
MOU_9Aug2017.DTA |  0270
WIN_10_25_15.DTA |  0270
WIN_10_29_15.DTA |  0270
WIN_11_01_15.DTA |  0270
WIN_11_07_15.DTA |  0270
WIN_11_13_15.DTA |  0270
Winous 14 Aug 15.DTA |  0270
Winous 14 Jul 15.DTA |  0270
Winous 14 Sep 15.DTA |  0270
Winous 15 Jul 15.DTA |  0270
Winous 27 Aug 15.DTA |  0270
Winous 7 Aug 15.DTA |  0270
ctonra:2017-01-03T19-41-53:mag_1jul16.dta |  0270
ctonra:2017-01-03T19-41-57:mou_1apr16_1.dta |  0270
ctonra:2017-01-03T19-42-00:mou_14oct16_1.dta |  0270
ctonra:2017-01-03T19-42-00:mou_15may16_1.dta |  0270
ctonra:2017-01-03T19-42-00:mou_18sep16.dta |  0270
ctonra:2017-01-03T19-42-01:mou_23aug16.dta |  0270
mou_04sep16.dta |  0270
mou_06aug16.dta |  0270
mou_12jul16.dta |  0270
mou_13apr16.dta |  0270
mou_13nov16.dta |  0270
mou_23oct16.dta |  0270
mou_24may16.dta |  0270
mou_27apr16.dta |  0270
mou_29mar16.dta |  0270
mou_29oct16.dta |  0270
mou_30sep16.dta |  0270
mou_31jul16.dta |  0270
mou_3apr16.dta |  0270
mou_6apr16.dta |  0270
mou_6jun16.dta |  0270
mou_7nov16.dta |  0270
mou_9may16.dta |  0270
winous9_22_15.DTA |  0270

**Comment by zcrysler** [comment]
Yes that looks right. 

I don't know how Lotek receivers work, when you say get them to enter a site code, do you mean somewhere within the receiver that will show up in raw data like we do with the gnomes?

**Comment by zcrysler** [comment]
There are also definitely instances where the site name was not changed in the deployment.txt file after a receiver was moved - will this affect how data is linked to project, or only in the case of duplicate receivers?

**Comment by jbrzusto** [comment]
Yes - the users of **Lotek**  receivers with duplicated serial numbers will need to set the
site code on the receiver itself, so that it always shows up as a specific value in the .DTA files
(i.e. this is something they do directly on the receiver, and is not done at motus.org)
Perhaps the best thing would be to adopt the same convention as for SGs:
add a "_1" to the serial number of the second receiver in each pair of duplicates.
So one would get site code "0000", the other would get site code "0001", and the data processing
server would automatically append "_1" to the serial number in the latter case.

For **SG**s, the shortlabel in `deployment.txt` is currently ignored by all processing **except** to distinguish between pairs of SGs with the same serial number.  And I'm only doing that for one pair so far (one of which is the SG at Pt. Lepreau) although I realize you've reported another pair of duplicated SG serial numbers, which I'll get to.

So for both receiver types, we'll be using a field originally intended to convey "site" info to instead extend
the serial number by one digit.
